requirements.txt

```diff
@@ -61,3 +61,7 @@ soupsieve==1.9
 typed-ast==1.3.1
 urllib3==1.24.1
 wrapt==1.11.1
+selenium==3.141.0
```

Comment:
Do we still need to have selenium as a requirement?

Reply:
No we do not use this anymore. I will remove it.